.travis.yml

```diff
@@ -28,7 +28,6 @@ env:
   - REQUIREMENTS=devel
 
 python:
-  - "3.5"
   - "3.6"
```

Comment:
Comment: We have actually discover (kudos to @zzacharo that its 3.6.2 the minimum). But I guess 3.6 is fine there. We can be more specific in end-user docs

Reply:
> We can be more specific in end-user docs

Ok. Travis is running Python 3.6.7 on Ubuntu 16.04.